### ISSUE TITLE
Plugin [id: 'com.expediagroup.graphql'] was not found in any of the following sources

### ISSUE THREAD

**Issue by justvil** [ExpediaGroup/graphql-kotlin#1203]
Hi,

I'm trying to integrate the graphql-kotlin lib into my Android app, but having trouble on the initial setup step. I used https://github.com/ExpediaGroup/graphql-kotlin/blob/master/examples/client/gradle-client/build.gradle.kts as my reference file, but after adding all the required fields I'm getting the following error:

```
org.gradle.internal.exceptions.LocationAwareException: Build file '.../mobileapp/android/app/build.gradle.kts' line: 5
Plugin [id: 'com.expediagroup.graphql'] was not found in any of the following sources:

- Gradle Core Plugins (plugin is not in 'org.gradle' namespace)
- Plugin Repositories (plugin dependency must include a version number for this source
Caused by: org.gradle.api.plugins.UnknownPluginException: Plugin [id: 'com.expediagroup.graphql'] was not found in any of the following sources:
- Gradle Core Plugins (plugin is not in 'org.gradle' namespace)
- Plugin Repositories (plugin dependency must include a version number for this source)
```

It's probably related to the version number, but I don't see a version number declared in your sample file. I'm probably missing something obvious here...

Thank you for your help!

**Comment by dariuszkuc** [comment]
Hello 👋 
We are using composite build (example project is a composite build depending on the library project) so it automatically points to the current version. When you integrate it to your project you have to specify the plugin version.

See https://opensource.expediagroup.com/graphql-kotlin/docs/plugins/gradle-plugin-tasks

**Comment by justvil** [comment]
Thank you @dariuszkuc ! Gradle is now able to locate the Plugin correctly.

By the way, since the task graphqlGenerateClient requires a task named compileKotlin to configure itself, does it mean that for Android it's only to perform the configuration explicitly? (Android projects do not contain a compileKotlin task, but compileDebugKotlin, etc.).

Thanks again!

**Comment by dariuszkuc** [comment]
Indeed we look explicitly for `compileKotlin` task (https://github.com/ExpediaGroup/graphql-kotlin/blob/master/plugins/graphql-kotlin-gradle-plugin/src/main/kotlin/com/expediagroup/graphql/plugin/gradle/GraphQLGradlePlugin.kt#L145). I'm unfamiliar with Android builds -> can you share sample project so I can take a look?

**Comment by justvil** [comment]
Hi @dariuszkuc , sure attaching a basic app project. I've already added the GraphQL Kotlin dependency and getting the compileKotlin error.

[GraphQLKotlinTestApplication.zip](https://github.com/ExpediaGroup/graphql-kotlin/files/6785772/GraphQLKotlinTestApplication.zip)

I also tried executing the tasks using the gradle command from a terminal, but it's not detecting the passed in parameter. I'm trying this:

`./gradlew -Pendpoint="https://n7b67.sse.codesandbox.io/graphql" graphqlDownloadSDL`

```
> Task :app:graphqlDownloadSDL FAILED

FAILURE: Build failed with an exception.

* What went wrong:
A problem was found with the configuration of task ':app:graphqlDownloadSDL' (type 'GraphQLDownloadSDLTask').
> No value has been specified for property 'endpoint'.
```

Thanks!
